common/src/test/java/tachyon/security/LoginUserTest.java

```diff
@@ -54,7 +54,7 @@ public void getSimpleLoginUserTest() throws Exception {
     User loginUser = LoginUser.get(conf);
 
     Assert.assertNotNull(loginUser);
-    Assert.assertFalse(loginUser.getName().isEmpty());
+    Assert.assertEquals(loginUser.getName(), System.getProperty("user.name"));
```

Comment:
is `user.name` used or checked anywhere in the code base, or does it depend on any pending PR? I could not search it in the current codebase.